views/keyword/_url_result_report.html

```diff
@@ -0,0 +1,15 @@
+<div class="search-url row">
+  <div class="search-url--result col-lg-6 col-sm-12">
```

Comment:
Instead of using Bootstrap utility class names directly, the team conventions is to use [SASS mixins](https://getbootstrap.com/docs/5.0/layout/grid/#mixins):

```
.search-url {
  @make-row()

  &__result {
    @make-col-span(6)
  }
} 
```